src/GitHub.App/ViewModels/PullRequestListViewModel.cs

```diff
@@ -138,10 +179,8 @@ public override void Initialize([AllowNull] ViewWithData data)
                 });
         }
 
-        void UpdateFilter(PullRequestState state, [AllowNull]IAccount ass, [AllowNull]IAccount aut)
+        void UpdateFilter(PullRequestState state, IAccount ass, IAccount aut)
```

Comment:
While I have no problems with naming a variable `ass` I think we should probably avoid swear words in case Microsoft runs their Policheck tool against this code. 😛